### ISSUE TITLE
'content-available' key within 'aps' dictionary

### ISSUE THREAD

**Issue by Illizian** [zendframework/ZendService_Apple_Apns#22]
I am using this library via the [laravel-push-notification](https://github.com/davibennun/laravel-push-notification/) package.

I'm having difficulty sending a push notification to an iOS device containing the key 'content-available' within the aps dictionary, as documented on Table 3-1 [here](https://developer.apple.com/library/ios/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/Chapters/ApplePushService.html).

It looks like the cause is in the [ZendService\Apple\Apns\Message\Alert](https://github.com/zendframework/ZendService_Apple_Apns/blob/master/library/ZendService/Apple/Apns/Message/Alert.php) class.

Is it just a case of adding the following:
```php
protected $contentAvailable;

public function setContentAvailable($value) {
    $this->contentAvailable = $value;
}

public function getContentAvailable() {
    return $this->contentAvailable;
}
```

If so, happy to submit a PR for this.

Cheers

**Comment by Ocramius** [comment]
@Illizian give it a try and provide a test case, I'd say.

**Comment by Illizian** [comment]
PR merged #23
